truffle/test/VaultNotaryTest.js

```diff
@@ -0,0 +1,384 @@
+const truffleAssert = require('truffle-assertions');
+const uuidv4 = require('uuid/v4');
+const uuidToHex = require('uuid-to-hex');
+
+const NotaryContract = artifacts.require("VaultNotary");
+
+
+
+function uuidToHex32(uuid) {
+    return uuid + '00000000000000000000000000000000'
+}
+
+
+contract('VaultNotary', async (accounts) => {
+    const OWNER = accounts[0];
+    const SHIPPER = accounts[1];
+    const CARRIER = accounts[2];
+    const MODERATOR = accounts[3];
+    const ATTACKER = accounts[9];
+
+    let contract;
+    before(async () =>{
+        contract = await NotaryContract.deployed();
+    });
+
+    async function registerVault() {
+        const vaultId = uuidToHex(uuidv4(), true);
+        const vaultUri = "uri";
+        const vaultHash = "hash";
+        await contract.registerVault(vaultId, vaultUri, vaultHash, {from: SHIPPER});
+        return vaultId;
+    }
+
+    //***************testing registerVault**************************************
+
+    it("should create a vault notary", async () => {
+        const vaultId = uuidToHex(uuidv4(), true);
+        const vaultUri = "uri";
```

Comment:
This is nitpicky - can we use 'real' vaultUri/vaultHash values? I know it doesn't affect these unit tests, but I like to test with realistic data if possible.